### ISSUE TITLE
NullReferenceException from GetMySqlDbType

### ISSUE THREAD

**Issue by MaceWindu** [mysql-net/MySqlConnector#845]
MySqlConnector 0.69.5

Similar to #812 but for geometrycollection type

```
System.NullReferenceException
   at MySqlConnector.Core.TypeMapper.GetMySqlDbType(String typeName, Boolean unsigned, Int32 length) in /_/src/MySqlConnector/Core/TypeMapper.cs:line 169
```

![image](https://user-images.githubusercontent.com/1099238/86740087-ac515600-c036-11ea-95f3-196ee3ff53e1.png)

Triggered when execute this [procedure](https://github.com/linq2db/linq2db/pull/2324/files#diff-9d72c269084f394097411c8259d7aaf3R532) with `SchemaOnly` command behavior

**Comment by MaceWindu** [comment]
FYI, `MySql.Data` also cannot handle this procedure

**Comment by bgrainger** [comment]
I can only reproduce with MySQL Server 8.0, not 5.7; not yet sure why it's server-dependent.

**Comment by bgrainger** [comment]
This is caused by MySQL Server 8.0 changing the standard alias for this data type to `GEOMCOLLECTION`: https://dev.mysql.com/worklog/task/?id=2388

**Comment by bgrainger** [comment]
Fixed in [0.69.6](https://github.com/mysql-net/MySqlConnector/releases/tag/0.69.6).
